test.txt

```diff
@@ -1 +1 @@
-and now there is text here. here is a test in my file . more text.
+here is a test in my file and here is a change
```

Comment:
great change!

Reply:
Thanks I worked really hard on this file.